protocol/Protocol.py

```diff
@@ -2667,15 +2676,24 @@ def in_ADMINBROADCAST(self, client, msg):
 		'''
 		self._root.admin_broadcast(msg)
 
-	def in_SETLATESTSPRINGVERSION(self, client, version):
+	def in_SETMINSPRINGVERSION(self, client, version):
 		'''
-		Set a new version of Spring as the latest.
+		Set a new min Spring version.
 
 		@required.str version: The new version to apply.
 		'''
-		self._root.latestspringversion = version
-		self.out_SERVERMSG(client, 'Latest spring version is now set to: %s' % version)
-
+		self._root.min_spring_version = version
+		self.in_BROADCAST(client, 'New engine version: Spring %s' % version)
+		
+		legacyBattleIds = []
```

Comment:
why this? self.broadcast_RemoveBattle(battle) can be called directly?!

Reply:
You can't remove elements from a dictionary while iterating through it.